src/info-notification.vala

```diff
@@ -51,37 +51,53 @@ public class IndicatorSound.InfoNotification: Notification
 
         /* Reset the notification */
         var n = _notification;
-        volume_label += "\n";
 
+        // FIXME: Uncomment this once we stop supporting Ubuntu 20.04
+        //bool bHints = notify_server_supports ("x-lomiri-private-synchronous");
+        bool bHints = AyatanaCommon.utils_is_lomiri ();
```

Comment:
As @peat-psuwit just let me know, the notification services have a get_capabilities DBus call that could be used to research what the service on the other side supports and what now. Maybe a follow-up approach

Reply:
I think it's `notify_server_supports()` that call `GetCapabilities()`. Does Ubuntu 20.04 uses NotifyOSD still, and thus also support `x-{lomiri,canonical}-private-synchronous`?

Reply:
`notify_server_supports` with any parameter crashes on Ubuntu 20.04. On newer systems it works fine.

Reply:
Hmm... That's weird, given that the code has been on UT 20.04 for a while...

Does the crash happen inside libnotify or somewhere else?